base/Dockerfile.rhel8

```diff
@@ -28,6 +28,11 @@ ARG BUILD_NUMBER=-1
 LABEL io.confluent.docker.build.number=$BUILD_NUMBER
 
 LABEL maintainer="tools@confluent.io"
+LABEL vendor="Confluent"
+LABEL version=$COMMIT_ID
```

Comment:
Do we actually use the $COMMIT_ID?

Reply:
your right, it should be GIT_COMMIT, not sure why we have commit id in there then. i will remove it.